test-case/check-sof-logger.sh

```diff
@@ -71,7 +71,8 @@ sof_alsa_card_found()
     # note: assumes SOF card names to start with "sof", e.g.
     #   - /proc/asound/sofsoundwire/id
     #   - /proc/asound/sofhdadsp/id
-    test -e /proc/asound/sof*/id
+    test -e /proc/asound/sofsoundwire/id ||
+        test -e /proc/asound/sofhdadsp/id
```

Comment:
The comment above does not make any sense after this change and must be removed (as shown in https://github.com/thesofproject/sof-test/issues/1243#issuecomment-2549132322)